src/api/config/routes/api_routes.rb

```diff
@@ -1,4 +1,4 @@
-require_relative 'routes_helper'
+require File.join(Rails.root, 'config', 'routes', 'routes_helper')
 
 OBSApi::Application.routes.draw do
```

Comment:
This shouldn't be needed since it's already in `config/routes.rb`. Same thing for `webui_routes.rb`.

I tried it and it works.